requirements.txt

```diff
@@ -19,3 +19,6 @@ Cython==0.27
 pydensecrf==1.0rc3
 tensorflow-gpu==1.10.0
 bokeh
+selenium
```

Comment:
@mkulariya1 we are not using `selenium` right?

Reply:
yes now we are not using it, will remove it.